xml/System.Buffers/StandardFormat.xml

```diff
@@ -90,9 +103,15 @@
       </Parameters>
       <Docs>
         <param name="other">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <summary>Indicates whether the current instances is equal to another <see cref="T:System.Buffers.StandardFormat" /> instance.</summary>
+        <returns><see langword="true" /> if the two instances are equal; otherwise, <see langword="false" />.</returns>
+        <remarks>  
+          <format type="text/markdown"><![CDATA[  
```

Comment:
should this remark be part of the main page instead of the individual overloads?